src/content/docs/style-guide/quick-reference/titles.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Titles vs Short titles vs Path vs Redirects
+---
+
+This doc explains the difference between Titles and Short titles. It also delves into how these are affected and affect Paths and Redirects.
+
+## Titles
+Titles are the title rendered at the top of each doc. The title of this doc is **Titles vs Short Titles vs Path vs Redirects**. 
+
+The title of a document is define in the front matter. This doc's title is defined below:
+
+```
+---
+title: Titles vs Short Titles vs Path vs Redirects
+---
+```
+
+
+## Short titles
+Short titles are displayed in the left nav. These are used in cases where the title is a bit too long to be easily scanned. The Short title of this doc is **Titles vs Short titles**.
+
+The Short title is defined in the corresponding navigation file. This doc's short title is define in `src/content/nav/style-guide.yml`:
+
+```
+- title: Tables
+  path: /docs/style-guide/quick-reference/tables
+```
+
+Read more about [editing the left nav](/docs/style-guide/processes-procedures/update-left-navigation-pane).
+
+## Paths and Redirects
+The path and file name of a specific doc has no bearing on the title of that doc. The path for this doc is `/docs/style-guide/quick-reference/tables`. Notice that although the file is named `tables`, neither the title or the short title is called `Titles`. 
```

Comment:
s/be "have" not "has" if path and file name are 2 separate things
I don't think this doc is called tables.
also "or" s/be "nor" although that does sound a bit pompous :-)